.gitignore

```diff
@@ -20,8 +20,10 @@
 .env.production.local
 tmp-storage
 tmp
-conductor-config.toml
 holofuel-conductor-config.toml
+example-conductor-config.toml
```

Comment:
Don't think we should be ignoring the `example-conductor-config`. We want that in the repo as an example.

Reply:
We had removed this file, but I added it back when comparing the manual conductor with the nix auto-generated one, so I just added the file to the ignore list.  At any rate, I still agree that it's unnecessary overall, so I'll go ahead and remove this as we shouldn't really be needing it beyond the local comparisons I was testing out.